packages/core/src/lib/TwaManifest.ts

```diff
@@ -528,6 +543,10 @@ export interface TwaManifestJson {
   };
   enableSiteSettingsShortcut?: boolean;
   isChromeOSOnly?: boolean;
+  isMetaQuest?: boolean; // Older Manifests may not have this field.
+  scope?: string; // Older Manifests may not have this field.
+  fullScopeUrl?: string; // Older Manifests may not have this field.
+  minSdkVersion?: number; // Older Manifests may not have this field.
```

Comment:
@andreban Please check these 4 lines above. Do they really have to be optional?

Reply:
They do, since those are new fields and won't exist in old manifests. This forces us to do proper checks for `undefined` later in the code.